lib/api/v1/serializers/users.go

```diff
@@ -0,0 +1,16 @@
+package serializers
+
+type UserResponse struct {
```

Comment:
We are using JSON:API for responses in almost projects, should we support it as well?
https://github.com/google/jsonapi

Reply:
Ahh actually I forgot to remove this, we didn't use it anywhere yet. But I agree that we should use the [`jsonapi`](https://github.com/google/jsonapi) I'll add it when we need to response with resources with ID 🙌🏼